data/addFieldsToCollection.js

```diff
@@ -32,6 +34,25 @@ async function addFieldsToCollection(model) {
     .catch((err) => console.log(err));
 }
 
+async function addPostReqsToSISCourse(model) {
```

Comment:
do you ever use this? Also what is 'model'

Reply:
yea we do use it. 
Its pretty much a copy of addFieldsToCollection but specifically for the sisCourse post reqs changes. When I tried running addFieldsToCollection on SISCourse, it didn't work, and I think it was because postReqs is a subfield of the versions field but I might be wrong/ called the function wrong. But, the new function worked fine.
Model is there since its a copy paste of addFieldsToCollection, Ill specify it has to be a SISCourse.